src/wizard/wizard-form.tsx

```diff
@@ -43,6 +44,14 @@ export default function WizardForm({
   const isLastStep = activeStepIndex >= steps.length - 1;
   const skipToTargetIndex = findSkipToTargetIndex(steps, activeStepIndex);
   const isMobile = useMobile();
+  const stepHeaderRef = useRef<HTMLDivElement>(null);
+
+  useEffectOnUpdate(() => {
+    if (stepHeaderRef && stepHeaderRef.current) {
+      const headers = stepHeaderRef.current.querySelectorAll('h1');
```

Comment:
Can't we use ref to target `InternalHeader` directly (rather than traverse through the DOM)?

Reply:
good catch, thank you! I missed `InternalHeader` ref

Reply:
I would change the selector to be `stepHeaderRef.current.querySelector('[tab-index]')` to not rely on tag name to which the tab-index is attached. Also, no need to query all as we only use one anyways.